docs/cmake.md

```diff
@@ -50,6 +50,57 @@ depending upon which variables are enabled/disabled.
 
+### Manually Creating config.cmake
+
+In order to work with config.cmake it is important to understand the purpose
+of its contained variables. Below is a list of the important variables with
+some information on their purpose.
+
+#### Optional OpenDDS Features
+When OpenDDS is built using the MPC build-system
+and optional features are specified, various C/C++; tao_idl; and opendds_idl
+compiler macros are set on dependent libraries or executables. These defines
+are set by MPC using the [dcps_optional_features.mpb](../MPC/config/dcps_optional_features.mpb)
+file.
+
+CMake-Based targets must also inherit the defined macros, so the functionality
+in `dcps_optional_features.mpb` is simulated using `options.cmake`
+and its associated `config.cmake`. As such, if features are enabled/disabled when
+compiling OpenDDS then it is paramount that the MPC features are also matched
+within `config.cmake` prior to compilation.
+
+This table outlines the feature-related variables and their _expected_ default
+values (generated when the configure script is run without arguments).
+If any feature is explicitly enabled/disabled in the OpenDDS build then the corresponding
+CMake variable below should be set to either _ON_ or _OFF_ depending on the desired state.
+
+| CMake Variable                   | Notes (from configure script output)    | Default |
+|---                               | ---                                     | ---     |
+|`OPENDDS_BUILT_IN_TOPICS`         | Built-in Topics                         | ON      |
+|`OPENDDS_CONTENT_FILTERED_TOPIC`  | ContentFilteredTopic (CS Profile)       | ON      |
+|`OPENDDS_CONTENT_SUBSCRIPTION`    | Content-Subscription Profile            | ON      |
+|`OPENDDS_MULTI_TOPIC`             | MultiTopic (CS Profile)                 | ON      |
+|`OPENDDS_OBJECT_MODEL_PROFILE`    | Object Model Profile                    | ON      |
+|`OPENDDS_OWNERSHIP_KIND_EXCLUSIVE`| Exclusive Ownership (Ownership Profile) | ON      |
+|`OPENDDS_OWNERSHIP_PROFILE`       | Ownership Profile                       | ON      |
+|`OPENDDS_PERSISTENCE_PROFILE`     | Persistence Profile                     | ON      |
+|`OPENDDS_QUERY_CONDITION`         | QueryCondition (CS Profile)             | ON      |
+|`OPENDDS_SECURITY`                | DDS Security plugin                     | OFF     |
+
+#### Build-Related Options
+
+The following values impact the build in one way or another.
+
+| CMake Variable            | Notes                                                              | Default |
+|---                        | ---                                                                | ---     |
+|`OPENDDS_ACE`              | Location of ACE root dir.                                          | N/A     |
+|`OPENDDS_TAO`              | Location of TAO root dir.                                          | N/A     |
+|`OPENDDS_STD`              | Forces C++ standard (Unix only)                                    | N/A     |
```

Comment:
The thought was that if the user specifies a C++ standard it should match the standard that was used to build OpenDDS. Otherwise potential issues may come about.

I will update the document to clarify!

Reply:
I should note that the reason this is specified in the config.cmake is just because that's how configure can relay user-specified info to the CMake build.

Reply:
Alright, yeah that makes sense, I didn't think about the standards being different.